lib/standards/aria-roles.js

```diff
@@ -17,21 +18,21 @@
 */
 const ariaRoles = {
   alert: {
-    type: 'widget',
+    type: 'structure',
     // Spec difference: Aria-expanded removed in 1.2
     allowedAttrs: ['aria-expanded'],
     superclassRole: ['section']
   },
   alertdialog: {
-    type: 'widget',
+    type: 'window',
```

Comment:
`window` isn't a type we support. [Valid types](https://github.com/dequelabs/axe-core/blob/develop/doc/standards-object.md#structure-1) are:

* abstract
* widget
* structure
* landmark

So probably this should be `structure` since it's not a landmark. Also according to https://www.w3.org/WAI/ARIA/1.2/class-diagram/rdf_model.svg `alert` is a section, which is under structure. 

```suggestion
    type: 'structure',
```

Reply:
Despite it being an abstract role under https://www.w3.org/TR/wai-aria/#abstract_roles referencing `alertdialog` and `dialog` under https://www.w3.org/TR/wai-aria/#window_roles ?

Reply:
window is fine